### ISSUE TITLE
When diff include chinese or japanese or korean, diff line may overflow

### ISSUE THREAD

**Issue by gitsang** [banga/git-split-diffs#25]
![image](https://user-images.githubusercontent.com/49085035/128802651-0d69ad12-384f-4ec3-94fb-a871b08faa1c.png)

I think it's because C/J/K letter occupies two bytes.

Is there any way to fix it?

**Comment by banga** [comment]
Yes, same as #11. The main issue is that we need to switch to something that implements grapheme-aware string lengths (https://unicode.org/reports/tr29/#Grapheme_Cluster_Boundaries). I haven't gotten around to it because it's a fairly involved change, but thanks for the nudge.

**Comment by banga** [comment]
Hi @gitsang, I just pushed a change to handle wide characters. It should be in the v0.15 release. Can you try it out and let me know if there are any remaining issues?

**Comment by gitsang** [comment]
Cool, it fixed now, thx.  👍
